api/package.json

```diff
@@ -1,25 +1,23 @@
 {
-  "name": "api",
+  "name": "language-partner-api",
   "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
+  "main": "server.js",
   "scripts": {
-    "dev-api": "nodemon dev.js",
-    "start": "node server.js",
-    "test": "jest --collectCoverage=true",
-    "test:watch": "jest --watch --collectCoverage=true"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon server.js"
```

Comment:
you can add these lines to your scripts

"dev-api": "nodemon dev.js",	    "test": "echo \"Error: no test specified\" && exit 1",
    "start": "node server.js",	    "start": "nodemon server.js"
    "test": "jest --collectCoverage=true",	
    "test:watch": "jest --watch --collectCoverage=true"